src/com/zone5cloud/core/terms/Terms.java

```diff
@@ -0,0 +1,10 @@
+package com.zone5cloud.core.terms;
+
+public class Terms {
+	private Terms() { }
+	
+	public static final String REQUIRED = "/rest/auth/terms/required";
```

Comment:
Should the url base "/rest/auth/" be constant? Not a big deal and happy to leave as is...